### PR TITLE
Penguin strategy pairs added

### PR DESCRIPTION
**Pull request by AbigailTCameron**

### CODE REVIEW

**Comment by timbotro** [comment]
This branch was really old so i rebased it to latest master

**Comment by timbotro** [comment]
All of the strategies are called PenguinStrategyAsset1Asset2 e.g. `PenguinStrategyPefiLink` which is not part of standard naming scheme. This will make the test logic quite brittle if we had to add a special case just for this one exchange type, as currently we expect it in the format `{ContractType}{Exchange}{Asset1}{Asset2}`

File: contracts/interfaces/masterchefIgloosv2.sol [hunk 1]
```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.6.12;
+
+import "../lib/erc20.sol";
+import "../lib/reentrancy-guard.sol";
+import "../lib/ownable,sol";
```
Comment (timbotro) [inline contracts/interfaces/masterchefIgloosv2.sol:5]: typo on here

File: contracts/interfaces/masterchefIgloos.sol [hunk 2]
```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.6.12;
+
+import "../lib/erc20.sol";
+import "../lib/reentrancy-guard.sol";
+import "../lib/ownable,sol";
```
Comment (timbotro) [inline contracts/interfaces/masterchefIgloos.sol:5]: typo on here

File: contracts/interfaces/igloomaster.sol [hunk 3]
```diff
@@ -0,0 +1,1458 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.6;
```
Comment (timbotro) [inline contracts/interfaces/igloomaster.sol:2]: Is this right? Most of our codebase is 0.6.X and this is only an interface after all

**Comment by timbotro** [comment]
Lots of compilation errors on this, probably typos. This branch needs a fixup before the next PR:
image

**Review by timbotro — CHANGES_REQUESTED** [review]
Lots of compilation errors on this, probably typos. This branch needs a fixup before the next PR:
image

**Comment by bigwampa** [comment]
no longer accurate/up-to-date

**Pull request closed without merging**